test/e2e/app-dir/dynamic-io/dynamic-io.test.ts

```diff
@@ -20,6 +20,12 @@ describe('dynamic-io', () => {
     expect(next.cliOutput).not.toMatch('Error occurred prerendering page')
   })
 
+  if (!isNextDev) {
+    it('should not warn about potential memory leak for even listeners on AbortSignal', async () => {
+      expect(next.cliOutput).not.toMatch('MaxListenersExceededWarning')
```

Comment:
This can make it flaky to run partial tests since you're not actually testing a specific test.